### PR TITLE
[ADD][16.0] product_supplierinfo_packaging

### PR DESCRIPTION
**Pull request by benwillig**
This new module allows to use packaging on a supplier info. The mechanism is the same as the purchase order, when we assign a minimum quantity, the packaging quantity is computed and vice versa.

### CODE REVIEW

**Comment by rousseldenis** [comment]
@benwillig Thanks for this. Is it a split from this https://github.com/OCA/stock-logistics-warehouse/tree/10.0/purchase_packaging ?

**Comment by benwillig** [comment]
@rousseldenis No it's not a split. I wasn't aware of this module but yes there could be submodules to implement the rest of missing features from purchase_packaging.

**Review by jbaudoux — APPROVED** [review]
Thanks. Looks useful module !

**Comment by benwillig** [comment]
@rousseldenis I just remembered I also did this one https://github.com/OCA/purchase-workflow/pull/2188

File: product_supplierinfo_packaging/models/product_supplierinfo.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+class ProductSupplierInfo(models.Model):
+    _inherit = "product.supplierinfo"
+
+    packaging_id = fields.Many2one(
+        comodel_name="product.packaging",
+        ondelete="restrict",
+    )
+    packaging_qty = fields.Float(
+        compute="_compute_packaging_qty",
+    )
```
Comment (lmignon) [inline product_supplierinfo_packaging/models/product_supplierinfo.py:19]: Why not a related?  `related="packaging_id.qty"`

**Review by lmignon — CHANGES_REQUESTED** [review]
A useful one. Thank you @benwillig A little comment.

File: product_supplierinfo_packaging/models/product_supplierinfo.py (hunk 1 above)
Comment (benwillig) [reply product_supplierinfo_packaging/models/product_supplierinfo.py:19]: Only for inheritance purpose

**Pull request closed without merging**